python/paddle/fluid/layers/nn.py

```diff
@@ -7455,6 +7456,58 @@ def maxout(x, groups, name=None):
     return out
 
 
+def reorg(x, stride, name=None):
+    """
+    Gives a stride to reorg the input tensor
+
+    Here are some example:
+
+    input is 4D LoDtensor with shape [batch, channel, height, width] and has an attrs stride = 2
+
+    reorg will do some math work to reorder the elements of input according to stride to construt
+    put with shape [batch, channel * stride * stride, height/stride, width/stride]
+
+    reorg is used to reorgnization the output of pre_layer and change the tensor to fit the shape
```

Comment:
This sentence doesn't seem to contain information. Can you explain in more detail?

Reply:
done in latest commit